object_manager.go

```diff
@@ -74,6 +76,7 @@ type IBObjectManager interface {
 	UpdateNetworkContainer(ref string, setEas EA, comment string) (*NetworkContainer, error)
 	UpdateNetworkView(ref string, name string, comment string, setEas EA) (*NetworkView, error)
 	UpdatePTRRecord(ref string, netview string, ptrdname string, name string, cidr string, ipAddr string, useTtl bool, ttl uint32, comment string, setEas EA) (*RecordPTR, error)
+	UpdateTXTRecord(ref string, recordName string, text string, useTtl bool, ttl uint32, comment string, eas EA) (*RecordTXT, error)
```

Comment:
The same about arguments' ordering as for CreateTXTRecord().

Reply:
Updated so arguments for TXT record match the A Record order throughout the code base